scripts/dev/e2e.py

```diff
@@ -69,18 +71,6 @@ def _prepare_test_environment(config_file: str) -> None:
     )
```

Comment:
I think for clarity the print statement should say "Creating Cluster Role Binding", similar for the role. Similarly the comments should reflect that and should also be corrected to say test pod instead of test runner. Happy to do this in a later PR (if wanted at all), just wanted to bring it up :)